Content.Shared/GameObjects/Components/Weapons/Melee/MeleeWeaponAnimationPrototype.cs

```diff
@@ -35,8 +36,8 @@ public void LoadFrom(YamlMappingNode mapping)
             var serializer = YamlObjectSerializer.NewReader(mapping);
 
             serializer.DataField(ref _prototype, "prototype", "WeaponArc");
-            serializer.DataField(ref _state, "state", null);
-            serializer.DataField(ref _id, "id", null);
+            serializer.DataField(ref _state, "state", string.Empty);
+            serializer.DataField(ref _id, "id", string.Empty);
```

Comment:
Does the client handle null for these? If so make them that.

Reply:
Quite sure it does not, will keep it like this.